src/components/Header/components/Links.tsx

```diff
@@ -31,13 +31,13 @@ export function MenuLinks({ data, onClick }: { data: HeaderData; onClick?: () =>
     );
 }
 
-const LINKS = ['about', 'events', 'sponsors', 'contact'];
+const LINKS = ['about', 'events', 'sponsors', 'contact', 'open source'];
 export function Links({ onClick }: { onClick?: () => void }) {
     return (
         <>
             {LINKS.map((link, i) => (
                 <Link
-                    href={`/${link}`}
+                    href={link === 'open source' ? '/open-source' : `/${link}`}
```

Comment:
is it not possible to change the open source link in `LINKS` to `open-source`?

Reply:
So, is it fine to have it show as "Open-Source" in the header?

Reply:
I think we should refactor it to have `LINKS` be `title: link` - it would be cleaner and more maintainable that way.